lib/common/http2_client.go

```diff
@@ -38,14 +43,29 @@ func NewHTTP2Client(timeout, idleTimeout time.Duration, keepAlive bool) (client
 		return
 	}
 
-	client = &HTTP2Client{
-		client: http.Client{
-			Transport: transport,
-			Timeout:   timeout,
-			CheckRedirect: func(req *http.Request, via []*http.Request) error {
-				return http.ErrUseLastResponse // NOTE prevent redirect
-			},
+	var httpClient HTTPClient
+
+	hc := &http.Client{
+		Transport: transport,
+		Timeout:   timeout,
+		CheckRedirect: func(req *http.Request, via []*http.Request) error {
+			return http.ErrUseLastResponse // NOTE prevent redirect
 		},
+	}
+	httpClient = hc
+
+	if retry {
+		ec := pester.NewExtendedClient(hc)
+		ec.Concurrency = 1
```

Comment:
Can I configure this options for another purpose? For example, with syncer, retry options can be different.

Reply:
OK then, I will make this configurable.